model/map/src/main/java/org/keycloak/models/map/realm/MapRealmAdapter.java

```diff
@@ -1101,6 +1103,8 @@ public ComponentModel importComponentModel(ComponentModel model) {
             }
         }
 
+        model.setId(UuidValidator.validateAndConvert(model.getId(), "component",
+                Boolean.valueOf(Config.scope(MapStorageSpi.NAME).get("rejectInvalidIds"))));
```

Comment:
nitpick: my IDE notified me that "parseBoolean" would be sufficient here and would avoid boxing the boolean.

```suggestion
                Boolean.parseBoolean(Config.scope(MapStorageSpi.NAME).get("rejectInvalidIds"))));
```

Reply:
Ah nice catch. Changed, thanks!

Reply:
reviewed, please mark this one as resolved.